network-api/networkapi/mozfest/templates/fragments/hero/carousel_hero.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n wagtailimages_tags %}
 <div data-mozfest-hero-carousel>
-  <div class="tw-relative tw-pt-[120px] tw-flex tw-flex-col tw-justify-end{% if not page.banner_video %} tw-pb-6 medium:tw-pb-[120px]{% endif %}">
+  <div class="tw-relative tw-pt-[120px] tw-flex tw-flex-col tw-justify-end{% if not page.banner_video %} tw-pb-6 medium:tw-pb-[120px] {% endif %}">
```

Comment:
@fessehaye Just spotted this `tw-justify-end{% ...`. I think this could also lead to issues if this class is not used elsewhere because is might get purged is the curly braces are touching the class name.